src/com/owncloud/android/datamodel/ThumbnailsCacheManager.java

```diff
@@ -362,6 +378,51 @@ public static ThumbnailGenerationTask getBitmapWorkerTask(ImageView imageView) {
         return null;
     }
 
+    public static Bitmap addVideoOverlay(Bitmap thumbnail){
+        Bitmap playButton = BitmapFactory.decodeResource(MainApp.getAppContext().getResources(),
+                R.drawable.view_play);
+
+        Bitmap resizedPlayButton = Bitmap.createScaledBitmap(playButton,
```

Comment:
The problem is that I need the exact coordinates of the three points forming the triangle to compute the exact visual center of the play button (which is not the center of the image).
So therefore I had modified the original image and cropped all outer space of the play button.
If we do this on all play buttons, I guess it is working.